test/functional/services/browser.ts

```diff
@@ -171,40 +171,33 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
       xOffset?: number,
       yOffset?: number
     ): Promise<void> {
-      switch (this.browserType) {
-        case Browsers.Firefox: {
-          // Workaround for scrolling bug in Firefox
-          // https://github.com/mozilla/geckodriver/issues/776
+      if (this.isW3CEnabled) {
+        // Workaround for scrolling bug in Firefox
```

Comment:
Is this comment still relevant?

Reply:
Yes, it still and the issue won't be fixed until Webdriver W3C spec update: https://github.com/w3c/webdriver/issues/1005

Reply:
I did not add ` if (this.isFirefox) {..}` because we have problem with Chrome W3C mode as well. This function should be updated/fixed as part of #34963

Reply:
I guess what I mean is that it's not just a "bug in Firefox", but a bug in the w3c spec?

Reply:
I will push a small update